### PR TITLE
fix(chieffancypants.angular-hotkeys): add css to add information about hotkeys

### PR DESCRIPTION
**Pull request by davinkevin**

### CODE REVIEW

File: package-overrides/github/chieffancypants/angular-hotkeys@1.4.5.json [hunk 1]
```diff
@@ -1,12 +1,18 @@
 {
-  "main": "build/hotkeys",
+  "directories": {
+    "lib": "build"
```
Comment (guybedford) [inline package-overrides/github/chieffancypants/angular-hotkeys@1.4.5.json]: This property will cause breaks for existing users so it is not encouraged.

**Comment by guybedford** [comment]
:+1:

**Comment by davinkevin** [comment]
I don't understand why it will break for existing users ? 
For that library, all the required files are under build (css or js), so we can override it.

If you prefer, I can modify my PR to use the previous syntax, but I would like to understand :smile: 

Thanks

**Comment by guybedford** [comment]
Yes please do. It's because it alters the file / module name structure to no longer contain the `build` path, which will get pushed out to all existing users. Overrides already in use must be backwards compatible, at least until jspm 0.17.

**Comment by davinkevin** [comment]
Modification done :smile:

**Pull request merged**